### PR TITLE
Move `otelcol-config.yml` to root of repo

### PR DESCRIPTION
**Pull request by mic-max**
I think this is a better place for this file, let me know if you have any concerns or better suggestions

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]
the otelcollector directory can be deleted now right?

File: compose.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ services:
     image: otel/opentelemetry-collector
     command: [ "--config=/etc/otelcol-config.yml" ]
     volumes:
-      - ./src/otelcollector/otelcol-config.yml:/etc/otelcol-config.yml
+      - ./otelcol-config.yml:/etc/otelcol-config.yml
```
Comment (cijothomas) [inline compose.yml:25]: i think you can keep ./src part.

Comment (cijothomas) [reply compose.yml:25]: to avoid too many files in the root

**Comment by mic-max** [comment]
> the otelcollector directory can be deleted now right?

Ya it's empty, no real notion of deleting folders in Git though, empty folders unless noted in a `.gitkeep` are just ignored

File: compose.yml (hunk 1 above)
Comment (julianocosta89) [reply compose.yml:25]: I'm with @cijothomas in this one.
I'd rather keep the otelcol-config within the otelcol folder.

Any reason you would like to move to the root @mic-max ?

Comment (mic-max) [reply compose.yml:25]: It just felt like a more top level config similar to the docker compose file. I'm fine to close this PR though and leave as is too - just wanted to spur a little discussion :)

**Pull request closed without merging**